pubtools/_pulp/tasks/garbage_collect.py

```diff
@@ -62,6 +69,22 @@ def run(self):
 
         LOG.info("Temporary repo(s) deletion completed")
 
+        # Clear out old all-rpm-content
+        LOG.info("Start old all-rpm-content deletion")
+        arc_threshold = self.args.arc_threshold
+        criteria = Criteria.and_(
+            Criteria.with_unit_type(RpmUnit),
+            Criteria.with_field("cdn_published", Matcher.less_than(datetime.utcnow() - timedelta(days=arc_threshold))),
+        )
+        try:
+            arc_repo = self.pulp_client.get_repository("all-rpm-content").result()
+            deleted_arc = list(
+                arc_repo.remove_content(criteria=criteria)
+            )
+            LOG.info("Old all-rpm-content deletion completed %s", len(deleted_arc[0].units))
```

Comment:
Please avoid writing the code in a way which assumes exactly one task is returned by remove_content.  You should loop over the tasks, even if it seems like there's always just one.

Also, can we please log the actual units and not just the number of units?  I'd find that info useful.